### DISCUSSION TITLE
Shareable Bookmarks/Presets

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3055] [Feature Requests]
Currently presets are scoped to be Global, Role, or User. However it would be nice to have the ability to easily "share" a bookmark as a portable window into a dataset. I would imagine the easiest way to do this would be from a URL with query strings.

**Comment by Saissaken** [comment]
hi @benhaynes !

I know this discussion is kinda old, but I wanted to link 2 other discussions and an existing PR that would cover this use case:

#16228
#6493

And the possible PR: #14081

I'd love to help in design/code to get this merged!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
